libs/hacl/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ if(MSVC)
     add_custom_command(
         TARGET ${HACL_CPP_TARGET_NAME} POST_BUILD
         COMMAND ${CMAKE_COMMAND} -E copy_directory
-        ${hacl_BINARY_DIR}/Release
+        ${hacl_BINARY_DIR}/${CMAKE_BUILD_TYPE}
```

Comment:
one side effect of the new paradigm is that you will get DEBUG encryption binaries when you run debug the UI app, which will slow things down like accumulations, etc. we can revisit this and enable the ability to load release encryption libs when debugging but i thought it was out of scope for what im doing here